s2i/assemble

```diff
@@ -25,7 +45,7 @@ go get -v ./...
 
 echo "---> Building application source..."
 go build -v -o main
-go install main
+cp ./main $GOBIN
```

Comment:
Try to run ”go install main” inside “github.com/openshift-s2i/s2i-go/1.6/test/test-app” folder – you get error.
Here is my output:
C:\Users\amsokol\Development\src\github.com\amsokol\s2i-go\1.6\test\test-app>go install main
can't load package: package main: cannot find package "main" in any of:
C:\Users\amsokol\Development\bin\go\src\main (from $GOROOT)
C:\Users\amsokol\Development\src\main (from $GOPATH)

The reason is “go install” expects full package name as input parameter:
go install github.com/amsokol/s2i-go/1.6/test/test-app

So it’s OK – we can use the following command:
go install $ROOT_PACKAGE

But we get another issue. The app name that is created in $GOBIN folder is not “main”. It’s “test-app”.
It means we need to parse $ROOT_PACKAGE to get folder name where main.go is located (“test-app” is our case).
((((

That’s why I’m using “go build -v -o main” + “cp ./main $GOBIN”